### DISCUSSION TITLE
Plotman, rsync and archiving

### DISCUSSION THREAD

**Discussion by XIII39** [ericaltendorf/plotman#789] [General]
Question, sort of unrelated to plotman but I was hoping someone could help me out..

I've been using plotman and I'm really enjoying the archiving feature. Now I'm thinking about my life outside of Chia..
My main business is the world of graphics, I generate a ton of gif and jpeg files on a daily and I currently use inotify-tools to monitor a folder on my laptop. When a new file is saved to a folder I use the `inotifywait create` watcher to trigger an rsync transfer of the newly saved file to move to my NAS..

Because I generate so many files per day and name them chronologically I would like to create something similar to the plotman archiving. I have a bunch of 6TB drives on my NAS and I would love to be able to have rsync find space on an available drive to send the file without having to manually change the rsync dst when I feel like the drive might be getting full.

I've been hunting though the rsync docs to find a solution, anyone have any ideas? Is there a script for rsync that I can use?

**Comment by XIII39** [comment]
I found a little workaround, which I think is kind of hilarious but it works.
I'm using plotman! I save all of my .gif files as .plot files, plotman triggers the transfer to my NAS then I have an inotifywait script that renames the transferred file from .plot back to .gif.. haha!

**Comment by racergoodwin** [comment]
You could make your inotify script do the move for you rather than have 2 processes running at the same time.
It's how I used to do my plot archiving before plotman.
Great usage though!

**Reply by XIII39** [reply]
Yeah, the problem is that I move my files to a JBOD of 4TB and 6TB drives so I like the archiving feature plotman offers. I name all of my files chronologically so I don't necessarily care which drive the land on, I can always search for the file date if/when I need them. 

It's nice to not have to monitor which drive has available space before I trigger a bunch of moves. Plotman does that job for me. Ha! I'm sure I could have written a short little script for rsync but why do all that work when I can just re-purpose plotman ;)

This archiving technique would be a great feature to add to rsync though, it's brilliant!
